packages/terra-base/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* 'base' class added to Base component
```

Comment:
We need to update this changelog to reflect the data attribute

Reply:
Changed!